doc/ref_arch/kubernetes/chapters/chapter01.rst

```diff
@@ -7,7 +7,7 @@ Introduction
 The intention of this Reference Architecture is to develop a usable Kubernetes based platform for the Telecom operator
 community. The RA will be based on the standard Kubernetes platform where ever possible. This Reference Architecture
 for Kubernetes will describe the high level system components and their interactions, taking the
-:doc:`common/chapter00:goals and requirements` and mapping them to real-world Kubernetes (and related)
+:ref:`common/chapter00:overview` and mapping them to real-world Kubernetes (and related)
```

Comment:
If referring to the chapter and not a specific section then you don't need the ":overview"

Reply:
should be fixed now, replaced with doc